extensions/wikia/CreateNewWiki/js/CreateNewWiki.js

```diff
@@ -88,10 +88,17 @@
 							UserLoginFacebook.closeSignupModal();
 						};
 					}
+
+					self.transition('NameWiki', true, '+');
+
 					// Load facebook assets before going to the login form
-					$.loadFacebookAPI(function () {
-						self.transition('NameWiki', true, '+');
-					});
+					$.loadFacebookAPI()
+						.done(function () {
+							$('.fb-loaded').removeClass('hidden');
+						})
+						.fail(function () {
+							// you can do something here if you want
```

Comment:
Hehe, I was expecting this line to be replaced with code, but since it's not, let's remove the `.fail()` call.